### ISSUE TITLE
userId is not transmitted on FpUsers.verified

### ISSUE THREAD

**Issue by ChristianSch** [proshin-roman/finapi-java-client#100]
Relating to: https://github.com/proshin-roman/finapi-java-client/blob/6e222b6e01acd85ec6d64f67790f528df4a769da/src/main/java/org/proshin/finapi/user/FpUsers.java#L49

Fails with `org.proshin.finapi.exception.FinapiException: Unexpected response code: expected=200, actual=400`:
```
// finAPI is an internal object holding the referenced information
FpUsers users = new FpUsers(finAPI.getEndpoint(), finAPI.getToken());
User user = users.create(new FpCreateParameters().withEmail(getEmail()));

if (!users.verified(externalId)) {
    System.out.println("user is not verified");
}
```

**Comment by proshin-roman** [comment]
@ChristianSch thanks for your help. That's really a bug. I see you already fixed it - could you create a pull request please so that I can include the fix into the next release?

**Comment by ChristianSch** [comment]
There you go: https://github.com/proshin-roman/finapi-java-client/pull/102/commits/1b3c684cfb274c141310b4d62695098ea276ec24
Thought it was automatically transmitted. Seems it was not.

**Comment by proshin-roman** [comment]
@ChristianSch I've added a comment to the pull request. Could you look at it, please?

**Comment by proshin-roman** [comment]
@ChristianSch the pull request has been merged. Fixed version will be published as v1.60.0-SNAPSHOT in a few minutes. Thanks for your contribution! Please, consider if the issue can be closed now.
